functions/Test-DbaDatabaseOwner.ps1

```diff
@@ -89,6 +87,10 @@ function Test-DbaDatabaseOwner {
                 Stop-Function -Message "Failure" -Category ConnectionError -ErrorRecord $_ -Target $instance -Continue
             }
 
+            #Sets the default login to sa if -TargetLogin is not present
+            if(!($PSBoundParameters.ContainsKey('TargetLogin'))){
+                $TargetLogin = "sa"
+            }
```

Comment:
This is already done now on now line 94... The `Test-Bound` is an internal command that validates if a parameter was passed in or not. When it uses the `-Not` it says to return `$true` if it is not passed in.